shell/platform/darwin/macos/framework/Source/FlutterSurfaceManager.h

```diff
@@ -10,4 +10,19 @@
 
 - (uint32_t)glFrameBufferId;
 
+- (void)setLayerContent;
+- (void)setLayerContentWithIOSurface:(IOSurfaceRef*)ioSurface;
+
+- (void)BackTextureWithIOSurface:(CGSize)size
```

Comment:
Nits: 
use camel case for objc method names : `backTexture...`
Also, I'd recommend to rename the method as below:
```
-(void)backTextureWithIOSurface:(IOSurfaceRef *) ioSurface size:(CGSize) size backingTexture:(GLuint)texture fbo:(GLuint)fbo;
```

1. The parameter should match the description, so the first parameter should be the `ioSurface`
2. Starting from the 2nd parameter, the "with" is not necessary as it made the signature too verbose.

Reply:
Changed it to your suggestion